SConstruct

```diff
@@ -353,10 +353,16 @@ if selected_platform in platform_list:
     elif methods.using_clang(env):
         # Apple LLVM versions differ from upstream LLVM version \o/, compare
         # in https://en.wikipedia.org/wiki/Xcode#Toolchain_versions
-        if (env["platform"] == "osx" or env["platform"] == "iphone") and major < 10:
-            print("Detected Apple Clang version older than 10, which does not fully "
-                  "support C++17. Supported versions are Apple Clang 10 and later.")
-            sys.exit(255)
+        if (env["platform"] == "osx" or env["platform"] == "iphone"):
```

Comment:
You can remove the extra parentheses. (We do use them by mistake in some places in the buildsystem because C++ devs tend to put them everywhere, but it's not pythonic :))